utils/profile.ts

```diff
@@ -237,19 +240,33 @@ export const getLimitedProfileTask = (
               profile.servicePreferencesSettings
             );
 
-      return isSenderAllowedTask.bimap(
-        error =>
-          error.kind === "UNEXPECTED_VALUE"
-            ? ResponseErrorInternal(`Unexpected mode: ${error.value}`)
-            : ResponseErrorQuery(
-                "Failed to read preference for the given service",
-                error
-              ),
-        isAllowed => ({
-          isAllowed,
-          profile
-        })
-      );
+      return isSenderAllowedTask
+        .bimap(
+          error =>
+            error.kind === "UNEXPECTED_VALUE"
+              ? ResponseErrorInternal(`Unexpected mode: ${error.value}`)
+              : ResponseErrorQuery(
+                  "Failed to read preference for the given service",
+                  error
+                ),
+          isAllowed => ({
+            isAllowed,
+            profile
+          })
+        )
+        .map(_ => {
+          telemetryClient.trackEvent({
+            name: "api.messages.subscription.senderallowed",
```

Comment:
What if we rename the event name related to the specific api. For example `api.limitedprofile.sender-allowed`

Reply:
Fine for me, I struggled a bit about the name